proto/src/prost/COSMOS_IBC_COMMIT

```diff
@@ -1 +1 @@
-4e9752d10cecb9c39067b156e14b3797d0523afe
+9d79e86da4a8efa2c8bc3af7621adf7aa2c6525a
```

Comment:
Hmm do we need to keep under version control 3 files for each of SDK and IBC module? I think we have

- proto/src/prost/COSMOS_IBC_COMMIT
- proto/src/prost/no_std/COSMOS_IBC_COMMIT
- proto/src/prost/std/COSMOS_IBC_COMMIT

(and similar for SDK commits)

Reply:
Yeah I am not sure why we have these in `no_std` and `std` where the top-level one would be enough. I guess it's a byproduct or running `proto-compiler` in both the `std` and `no_std` folders. The top-level one instructs the `sync-protobuf.sh` as to which commits to checkout. I'll see if I can clean this up.

Reply:
https://github.com/informalsystems/ibc-rs/pull/1825/commits/c26cdf5b9c63b4218088b15d3f87f6d8d9c74a63